lib/services/gitlab/gitlab_resource.rb

```diff
@@ -0,0 +1,44 @@
+class GitlabResource < GenericResource
+
+    def prepare_request
+        super
+        #auth_header
+    end
+
+    #def auth_header
```

Comment:
Remove dead code (unless this method needs to be actually implemented -- haven't tested the code live yet)